Content.Shared/Clothing/Components/StealthClothingComponent.cs

```diff
@@ -0,0 +1,43 @@
+using Content.Shared.Actions;
+using Content.Shared.Actions.ActionTypes;
+using Content.Shared.Clothing.EntitySystems;
+using Robust.Shared.GameStates;
+
+namespace Content.Shared.Clothing.Components;
+
+/// <summary>
+/// Adds StealthComponent to the user when enabled, either by an action or the system's SetEnabled method.
+/// </summary>
+[RegisterComponent, NetworkedComponent, AutoGenerateComponentState(true), Access(typeof(StealthClothingSystem))]
+public sealed partial class StealthClothingComponent : Component
+{
+    /// <summary>
+    /// Whether stealth effect is enabled.
+    /// </summary>
+    [ViewVariables, AutoNetworkedField]
+    public bool Enabled;
```

Comment:
Datafield + vv readwrite or admins kill me.

Reply:
That's fine, most of the time they just dirty the entity anyway.

Reply:
ok real it did work